### ISSUE TITLE
[FEATURE IMPROVEMENT] Return a value other than 30 on retry if the answer is known to take longer

### ISSUE THREAD

**Issue by asant57** [usnistgov/ACVP-Server#116]
Suggest adding logic to send a retry value closer to when the actual result can be expected done. 

this would relieve some load on the server as the clients should sit and wait for the retry value provided.

**Comment by livebe01** [comment]
Hi @asant57, thank you for this suggestion. You may also be interested in the HealthCheck endpoint added in v1.1.0.18 and documented here: https://github.com/usnistgov/ACVP-Server/wiki/Health-Check---Server-Version-Endpoint

**Comment by asant57** [comment]
the API provided seems to be specifically for ACVP server health. whereas my envisioning was more of the regular retry:30 that is returned. for example if an algorithm, say DSA, is known to take 2 minutes to generate the keyGen request data the server could return 120 in the retry and reduce the number of queries a client makes just to get the retry by a factor of 4 (in this example).

**Comment by livebe01** [comment]
Yes, sorry. I didn't mean to imply that the HealthCheck was the same as what you are suggesting. Optimizing the retry value makes sense and is a valuable suggestion. I've marked this ticket as an "enhancement" and we'll consider it as we make improvements to the server code.
